src/components/leaderboard/TopThreeCard/index.tsx

```diff
@@ -0,0 +1,28 @@
+import styles from '@/components/leaderboard/TopThreeCard/style.module.scss';
+
+interface UserCardProps {
+  position: 1 | 2 | 3;
+  state: String;
+  rank: String;
+  name: String;
+  points: number;
+  image: String;
+}
+const positionNames = ['first', 'second', 'third'];
+
+const TopUserCard = ({ position, state, rank, name, points, image }: UserCardProps) => {
+  const trimName = name.length >= 25 ? `${name.substring(0, 24)}...` : name;
+  return (
+    <div className={`${styles.leaderboardCard} ${styles[positionNames[position - 1]]}`}>
```

Comment:
``` HTML
<div className={styles.leaderboardCard} data-position={positionNames[position - 1}>
```

```CSS
.leaderboardCard[data-position="first"] {
  background-color: theme gold
}
```
There's a different type of selector in standard HTML called a data attribute. We can use a class name for the styles that are common to all leaderboard card components but since the position is technically a changing property, we can define an attribute. It's very similar to our existing approach but it's a little bit more semantically correct because the position is a property of the leaderboard card that we're using to store some information.

For some reason, the dynamic bracket CSS module class names complain a lot in TypeScript so this approach should work instead.